### DISCUSSION TITLE
Reload custom activities without restarting ELSA server

### DISCUSSION THREAD

**Discussion by parikhpratik** [elsa-workflows/elsa-core#3290] [Design & tech]
I have one custom activity and it is also registered in ELSA engine.
Now I am updating that custom activity code and I want to upgrade that custom activity in ELSA engine and ELSA dashboard without restarting ELSA.
In ELSA 2.0 is there any way possible?

**Comment by tomy2105** [comment]
This has already been discussed in #1501 :).
